sketches/M2_CAN_haz_bus/defines.h

```diff
@@ -35,26 +35,6 @@
 #define CMD_CAN_SENDRECV_ISOTP   0x48
 
 /* constants for setting baudrate for the CAN bus */
-uint32_t baud_rates_table[19] = {
```

Comment:
If a variable is declared in a header file, trying to reference that variable by including the header file in another source file won't work due to the include guards. The correct (really, only) way to share a global variable in C is to declare the variable at file scope and put an extern reference to the variable in the header file.

This is due to C not having a truly global namespace.